.envrc

```
@@ -56,5 +56,6 @@ if [ ! -f "$DIRENV_INSTALLED_CRATES" ]; then
 	invoke deps.cargo.install
 fi
 
+export SCONSFLAGS="${SCONSFLAGS:+$SCONSFLAGS }-C $(pwd)"
```

```suggestion
export SCONSFLAGS="${SCONSFLAGS:-} --directory=$(expand_path .)"
```
Let's use what's available in [direnv-stdlib](https://direnv.net/man/direnv-stdlib.1.html) to make this code as portable as possible, and long options where possible to increase readability.